Cubical/Categories/Displayed/Constructions/BinProduct/More.agda

```diff
@@ -38,8 +51,22 @@ module _ {C : Category ℓC ℓC'} (D : Categoryᴰ C ℓD ℓD')
 
 -- Product within a fiber
 -- TODO: replace this with ∫ᴰ ?
--- Product of total categories
--- todo: replace with the next construction & reindexing?
+
+-- TODO: is there an easy way to reduce this code duplication?
```

Comment:
whats's duplicated here? The module structure and parameters?

If so, can't these anonymous modules be nested to avoid duplication?

Reply:
Oh this comment is outdated because I fixed the code duplication